src/routes/Home.jsx

```diff
@@ -0,0 +1,12 @@
+import AddnewBooks from '../components/AddnewBook';
+import Book from '../components/Book';
+
+const Home = () => (
+  <div className="books">
+    <Book title="Lolita" author="Vladimir Navokob" />
+    <Book title="Madamme Bovary" author="Gustave Flaubert" />
```

Comment:
- [ ] Good job working on this, but to make it easier to add more books later please kindly create a `BookList` component that renders a list of books using the `Book` component.